sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -43,6 +43,11 @@ public partial class AzureSasCredential
         public string Signature { get { throw null; } }
         public void Update(string signature) { }
     }
+    public partial class CertificateCredential
+    {
+        public CertificateCredential(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate) { }
+        public System.Security.Cryptography.X509Certificates.X509Certificate2 ClientCertificate { get { throw null; } }
```

Comment:
IMO, if you're goal is to avoid making breaking changes later if you ever support more types, calling this property simply `ClientCertificate` may be too general. Why not just `X509Certificate2`?